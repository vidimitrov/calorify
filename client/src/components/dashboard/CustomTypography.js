import Typography from '@material-ui/core/Typography';
import styled from 'styled-components';

const CustomTypography = styled(Typography)`
  flex-grow: 1;
`;

export default CustomTypography;
