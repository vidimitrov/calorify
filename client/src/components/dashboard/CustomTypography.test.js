import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import CustomTypography from './CustomTypography';

it('Dashboard/CustomTypography component renders correctly', () => {
  const tree = renderer.create(<CustomTypography />).toJSON();
  expect(tree).toMatchSnapshot();
});
