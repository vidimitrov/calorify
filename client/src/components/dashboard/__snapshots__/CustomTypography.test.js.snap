// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/CustomTypography component renders correctly 1`] = `
.c0 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

<p
  className="MuiTypography-root-1 MuiTypography-body1-10 c0"
/>
`;
